src/form-element-creator/element-creator.ts

```diff
@@ -1,17 +1,16 @@
 export class ElementCreator {
-
-    // TODO: zmienilam typ createElement na any zeby testy przeszly
-    createElement(tagName: string, attributes?: Record<string, string>): any {
-        if (typeof document !== 'undefined') {
-            const element = document.createElement(tagName);
-            if (attributes) {
-                Object.keys(attributes).forEach((key) => {
-                element.setAttribute(key, attributes[key]);
-            });
-        }
-        return element;
-        } else {
-            console.error('The document object not found.')
-        }
+  // TODO: zmienilam typ createElement na any zeby testy przeszly
+  createElement(tagName: string, attributes?: Record<string, string>): any {
+    if (typeof document !== "undefined") {
+      const element = document.createElement(tagName);
+      if (attributes) {
+        Object.keys(attributes).forEach((key) => {
+          element.setAttribute(key, attributes[key]);
+        });
+      }
+      return element;
+    } else {
```

Comment:
nie potrzebujesz else jezeli masz return w ifie powyzej. Po prostu zostaw sam console.error Nie trzeba wrapowac w else bo jezeli wejdzie w if to zrobi return i ten log sie nie wyswietli a jak nie wejdzie to z automatu go wyswietlisz tak samo jakbys to miala w else